test/JDBC/expected/constraint_table_usage-vu-verify.out

```diff
@@ -0,0 +1,58 @@
+select * from information_schema.constraint_table_usage where table_name like 'constraint_table_usage_vu_prepare_tb%' order by table_name,constraint_name,table_schema;
```

Comment:
The user that can access a table should be able to see its data in the view. Please add tests to verify the behaviour is correct in these cases.

Repro:

> User1:
create table test1 (a int primary key)
grant SELECT on test1 to User2
User2:
SELECT * FROM information_schema.CONSTRAINT_TABLE_USAGE
Should get the data here

Reply:
And also check that the user does not see rows corresponding to objects that they do not have privileges to

Reply:
The behavior in SQL Server and PG is to show information for all corresponding objects, regardless of privileges. I think it is best to keep the behavior consistent for now.

Reply:
We should use "setuser" to specify from which user the view should retrieve data through. The view will return all previous data without this clarification. However, "setuser" is not supported in Babelfish currently. In light of this, I have adjusted my "where" condition to include has_table_privilege() and has_any_column_privilege(). This is the currently accepted syntax in information schema views that have already been implemented.

Reply:
An additional testing file was added in the latest commit to check the "select" privilege.